packages/docs/src/shared-components/appshell.ts

```diff
@@ -26,17 +26,8 @@ export const createFooter = () => html`
       margin: 0;
     }
 
-    .link-wrapper {
-      display: flex;
-      gap: var(--syn-spacing-large);
-      justify-content: center;
-      list-style: none;
-      margin: 0;
-      padding: 0;
-    }
-
     @container (max-width: 768px) {
-      .link-wrapper {
+      .syn-link-list {
         flex-direction: column;
         gap: var(--syn-spacing-small);
```

Comment:
the gap can now be removed as the syn-link-list styles applies it

```suggestion
```

Reply:
Good catch, removed it.